default/customizations/custom_sitreps.txt

```diff
@@ -104,6 +104,33 @@ EffectsGroup
             ]
             empire = Source.Owner
 
+EffectsGroup
+    scope = And [
+        Capital
+        Not OwnedBy empire = Source.Owner
+        Number low = 1 condition = And [
```

Comment:
What does this Number condition do?

Reply:
For every empire [but the researching empire].. (Capital Not OwnedBy ..) 
which can detect an object owned by the researching empire (Number low ...)

But probably you usually know the detection strength of unsee empires as well. Does Olegs rule prevent knowing the detection strength?

Reply:
@agrrr3 No, empire detection strength is known with rules about tech and statistics enabled.

Reply:
Ok then this is not strictly necessary - i will remove it for the time being. It might be though that one usually is only interested in empires in reach. But if you do not have a lot of empires in the game i dont think this will spam too much.

Reply:
Please then add comment(s) saying the scope includes capital objects owned by empires other than the empire that researched the tech, if there is at least one object that is owned by the empire that researched the tech and the owner of the scope candidate can see.